lib/fastlane_core.rb

```diff
@@ -6,6 +6,7 @@
 
 # Third Party code
 require 'colored'
+require 'babosa'
```

Comment:
There were no other requires. So I though you were requiring all 3rd party here. Will update now.